packages/backend/tsconfig.json

```diff
@@ -2,5 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "build"
-  }
+  },
+  "exclude": ["build", "node_modules"]
```

Comment:
Why do we need to modify the original tsconfig?

Reply:
There is no reason to include build and node_modules when using typecheck or smth similiar.